### PR TITLE
Improve the targets prune help by adding examples

### PR DESCRIPTION
**Pull request by vkhoroz**
Signed-off-by: Volodymyr Khoroz <volodymyr.khoroz@foundries.io>

### CODE REVIEW

File: subcommands/targets/prune.go [hunk 1]
```diff
@@ -26,6 +26,21 @@ func init() {
 		Short: "Prune target(s)",
 		Run:   doPrune,
 		Args:  cobra.MinimumNArgs(1),
+		Example: `
+  # prune a single target name its name:
```
Comment (vanmaegima) [inline subcommands/targets/prune.go:30]:
```suggestion
  # prune a single target by its name:
```

Comment (doanac) [reply subcommands/targets/prune.go:30]: i think other parts of fioctl talk about `target name` and `target version`. So maybe we do `prune a single target name`

File: subcommands/targets/prune.go [hunk 2]
```diff
@@ -26,6 +26,21 @@ func init() {
...
+  fioctl targets prune intel-corei7-64-lmp-123
+
+  # prune all targets with the "custom.tags" set to ["devel"]:
+  fioctl targets prune --by-tag devel
+
+  # prune all targets with the "custom.tags" set to ["devel"] except for the last 10:
```
Comment (doanac) [inline subcommands/targets/prune.go:36]: I like "last 10" but i wonder if "most recent 10" would make people more comfortable? I don't have a strong opinion.

File: subcommands/targets/prune.go (hunk 1 above)
Comment (vkhoroz) [reply subcommands/targets/prune.go]: This is what happens when one spends less then 5 minutes on the commit.
Fixed in force-push.

File: subcommands/targets/prune.go (hunk 2 above)
Comment (vkhoroz) [reply subcommands/targets/prune.go]: Changed in force-push.

**Review by doanac — APPROVED** [review]
a could usage statement is more valuable than 1000 lines of backend validation improvements :)

**Pull request merged**